policies/AC-Access_Control/component.yaml

```diff
@@ -248,3 +331,148 @@
+      text: |-
+        A successful control response will document how, if wireless
+        connectivity is permitted, mobile devices are authorized to
+        connect to organizational networks (e.g. MDM).
+
+- control_key: AC-19 (5)
+  standard_key: NIST-800-53
+  covered_by: []
+  implementation_status: none
+  narrative:
+    - text: |-
+        A successful control response will document how either full-device
+        encryption on mobile devices is being used. This is only applicable
+        if mobile is within scope of your security package.
+
+- control_key: AC-20
+  standard_key: NIST-800-53
+  covered_by: []
+  implementation_status: none
+  narrative:
+    - key: a
+      text: |-
+        The organization will be responsible for establishing terms and conditions
+        allowing authorized individuals to access the organization application
+        from external information sysetms. A successful control response will
+        need to outline the terms and conditions and the external information
+        systems to which those terms and conditions apply.
+    - key: b
+      text: |-
+        The organization will be responsible for establishing terms and conditions
+        allowing authorized individuals to process, store, or transmit
+        customer-controlled information using external information systems. A
+        successful control response will need to outline the terms and
+        conditions and the external information systems to which those terms
+        and conditions apply.
+
+- control_key: AC-20 (1)
+  standard_key: NIST-800-53
+  covered_by: []
+  implementation_status: none
+  narrative:
+    - key: 'a'
+      text: |-
+        The organization will be responsible for verifying the implementation of
+        required security controls on external information systems used to
+        access the organization application. A successful control response will
+        need to address how this verification occurs (e.g. through an
+        independent assessment).
+    - key: 'b'
+      text: |-
+        If part (a) is not possible, then the organization will be responsible
+        for establishing and retaining approved information system connection
+        or processing agreements with the external entity hosting the external
+        information system used to access the organization application.
+
+- control_key: AC-20 (2)
+  standard_key: NIST-800-53
+  covered_by: []
+  implementation_status: none
+  narrative:
+    - text: |-
+        The organization will be responsible for restricting or prohibiting the
+        use of customer-controlled portable storage devices on external
+        information systems. If such use is not prohibited, a successful
+        control repsonse will need to address specific restrictions on how
+        or under what conditions such use may occur.
+
+- control_key: AC-21
+  standard_key: NIST-800-53
+  covered_by: []
+  implementation_status: none
+  narrative:
+    - key: 'a'
```

Comment:
@carlosmmatos might want to make sure that instances of `'` are removed.

Reply:
good call

Reply:
fixed